CIRC-5913 Updated IRONdb Grafana integration docs for hosted SaaS (#133)
circonus/docs: content/irondb/tools/irondb-grafana.md

1. Create a new datasource and select IRONdb from the `Type` drop down.

## HTTP

### URL

* Standalone: If this datasource will access a standalone IRONdb cluster, then this is the URL where IRONdb can be accessed. Example: `http://localhost:8112`

> * Hosted: If this datasource will access data via the hosted Circonus platform, this is the URL where the Circonus API can be accessed. Default: `https://api.circonus.com`

Reviewer: You have this marked "Default."  Is this the code default? e.g. it's pre-populated with this?   Another thing to consider here is that "Hosted" does not necessarily mean SaaS.   It could also be Inside customers. 
```suggestion
* Hosted: If this datasource will access data through a Circonus installation, this is the URL where the Circonus API can be accessed. Default: `https://api.circonus.com`
```
